src/components/AllCSS/addBook.css

```diff
@@ -0,0 +1,195 @@
+.book-form {
+  width: 75%;
+  height: 14%;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+  height: 140px;
+  margin: unset;
+}
+
+.action {
+  width: 3.063rem;
+  height: 1.125rem;
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-size: 1.175rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: var(--black-two);
+  padding: 5px;
+}
+
+.title {
+  width: 20.313rem;
+  height: 1.813rem;
+  font-weight: bold;
+  font-size: 25px;
+  font-stretch: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin: unset;
+  padding: 2px;
+}
+
+.author {
+  width: 20.688rem;
+  height: 1.188rem;
+  font-size: 1.295rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+  margin: unset;
+  padding: 2px;
+}

```

Comment:
- [ ] The same thing here, Kindly check my comment above ⬆ In additional to, Please "Don't repeat the same rule" 
> 💡 For more info, [Check this Article](https://speckyboy.com/good-bad-css-practices/)

> For Example you can group selectors ( , )
```css

.title,
.author {
  height: 1.813rem;
  margin: unset;
  padding: 2px;
}

.title {
  width: 20.313rem;
  font-weight: bold;
  font-size: 25px;
  letter-spacing: -0.2px;
  color: #121212;
}

.author {
  width: 20.688rem;
  font-size: 1.295rem;
  font-weight: 300;
  color: #4386bf;
}

```